uint16 estop_fr
uint16 estop_bl
uint16 sw_start
uint16 sw_stop
uint16 in_spare1
uint16 in_spare2